loginsightwebhookdemo/bugzilla.py

```diff
@@ -81,4 +88,10 @@ def bugzilla(ALERTID=None, TOKEN=None, PRODUCT=None, COMPONENT=None, VERSION=Non
             "summary" : a['AlertName'],
             "description": a['info'],
         }
-        return callapi(BUGZILLAURL + '/rest/bug?' + auth, 'post', json.dumps(payload), headers, None, check = VERIFY)
+        # op_sys and rep_platform may not be needed, but are to test on landfill
+        if "landfill.bugzilla.org" in BUGZILLAURL:
```

Comment:
That's a matching a substring anywhere in the `BUGZILLAURL`, which is not necessarily the hostname involved. It's a tiny edge-case, but it a bit wider of a net than intended.